### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by donnisnoni95**
I think those who read will think that it is support (such as contacts, chat and help).

We will make #support for the chat section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,11 +370,11 @@ This should do the trick. You can also check if there is any python at all via `
 Please remember `python2` is the one needed.
 
 
-#### known issues building on Windows
+#### Known issues building on Windows
 Right now there are few gotchas and the build will most likely fail. Please help with a PR if you know how to solve the issue, thank you!
 
 
-## Support
+## Features That Supported
```
Comment (romgrk) [inline README.md]: I think "Features that are supported" would be better

Comment (donnisnoni95) [reply README.md]: Yes... exactly.... sorry for my english.. :smile:

**Comment by donnisnoni95** [comment]
Done ! Great Work Guys... :smiley: :+1:

**Comment by romgrk** [comment]
You closed this PR, but it wasnt merged yet. You can push new commits to fix the grammar issue and then we can merge it.

**Comment by donnisnoni95** [comment]
Sorry, I told you already. I'm not very experienced yet. I'm still learning. 

I got wifi in my house just in 2 months, because the ISP had just reached our area. That's why I have been active on Github recently.

**Comment by romgrk** [comment]
Done, thanks!

**Pull request merged**